content/en/security/automation_pipelines/set_due_date.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Set Due Date Rules
+further_reading:
+  - link: "/security/automation_pipelines"
+    tag: "Documentation"
+    text: "Automation Pipelines"
+---
+
+{{< callout url="https://www.datadoghq.com/product-preview/security-automation-pipelines/" >}}
+  Automation Pipelines is in Preview. To enroll in the Preview for due date rules, click <strong>Request Access</strong>.
+{{< /callout >}} 
+
+Configure due date rules to ensure vulnerabilities are addressed within your specified SLO time frames. By setting these due dates, you can automate accountability, meet compliance requirements, and prioritize the prompt remediation of security issues, thereby preventing potential exploitation.
+
+Notification rules are pre-set conditions that automate security issue notifications for your team. With notification rules, you don't need to manually set up alerts for each individual detection rule. Instead, you can configure rules to cover various scenarios by specifying factors like severity, rule type, and tags.
+
+Notification rules help automatically alert your team about security problems. Instead of setting up alerts for each issue one by one, you use notification rules. You can set these rules to cover different situations by choosing options like severity, type, and specific tags.
+
+## Create a due date rule
+
+1. On the [Automation Pipelines][2] page, click **Add a New Rule** and select **Set Due Date**.
+1. Enter a descriptive name for the rule, for example, **Cloud Infrastructure Anomaly Warnings**.
+1. Use the following boxes to configure the rule criteria:
+    - **Any of these types**: The types of findings that the rule should check for. Available types include **Application Code Vulnerability**, **Application Library Vulnerability**, **Container Image Vulnerability**, **Misconfiguration**, **Attack Path**, **Identity Risk**, and **API Security Finding**.
```

Comment:
I know this list is repeated in the other rule topics, so this isn't a big deal by any means if you don't want to hold up publication to get this change in all of them. I just find nested lists a lot easier to read than trying to visually parse non-bolded commas between bolded inline list items.
```suggestion
    - **Any of these types**: The types of findings that the rule should check for. Available types include:
      - **Application Code Vulnerability**
      - **Application Library Vulnerability**
      - **Container Image Vulnerability**
      - **Misconfiguration**
      - **Attack Path**
      - **Identity Risk**
      - **API Security Finding**
```